tcf

```diff
@@ -285,6 +286,14 @@ def _cookies(args):
 
 if __name__ == "__main__":
     tcfl.tc.version = commonl.version_get(tcfl, "tcf")
+
+    if "TCF_NEW_COMMANDS" in os.environ:
+        tcfl.ui_cli.commands_old_suffix = "-old"
+        tcfl.ui_cli.commands_new_suffix = ""
+    else:
+        tcfl.ui_cli.commands_old_suffix = ""
+        tcfl.ui_cli.commands_new_suffix = "2"
```

Comment:
basically, if we have that variable in the env, we wont have to specify `2` in every command and if we want to use the old ones we add `-old`. This is cool, where should we have this documented?

Reply:
yap; I am hesitant on documenting it for the time being, more of a developer feature?